ws/src/main/webapp/WEB-INF/schemas/beans.xsd

```diff
@@ -38,6 +38,7 @@
         <sequence>
             <element name="label" type="beans:customStringType" minOccurs="0"/>
             <element name="secondaryLabel" type="beans:customStringType" minOccurs="0"/>
+            <element name="subjectOid" type="beans:customStringType"/>
```

Comment:
can you call this just "OID" to be consistent with naming convention of other fields?